File: 30 Days of Code/Day 6/LetsReview.cpp
#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
using namespace std;


int main() {
    /* Enter your code here. Read input from STDIN. Print output to STDOUT */   
    int T;
    string s;
    cin>>T;
    int index=0;
    for (int i=0;i<T;i++){
        cin>>s;
        for (index=0;index<s.length();index++){
            if (index%2==0) cout<<s[index];
        }
        cout<<" ";
        for (index=0;index<s.length();index++){
            if(index%2!=0) cout<<s[index];
        }
        cout<<"\n";
    }
    return 0;
}

File: 30 Days of Code/Day 6/LetsReview.py
# Enter your code here. Read input from STDIN. Print output to STDOUT
T=int(input())
i=0
for i in range(0,T):
    s=input()
    for j in range(0,len(s)):
        if j%2==0:
            print(s[j],end='')
    print(" ",end='')
    for j in range(0,len(s)):
        if j%2!=0:
            print(s[j],end='')
    print()




